state-chain/cf-integration-tests/src/new_epoch.rs

```diff
@@ -254,6 +254,7 @@ fn can_consolidate_bitcoin_utxos() {
 				crate::network::fund_authorities_and_join_auction(MAX_AUTHORITIES);
 
 			testnet.move_to_the_next_epoch();
+			witness_ethereum_rotation_broadcast(1);
```

Comment:
Why not add this into `move_to_the_next_epoch`? Does it not always need to be called?

Reply:
this is called on only on first rotation and not on every rotation so we cant move it inside.

Reply:
Sorry yes, I noticed that later in the review.